### PR TITLE
Ask mb read me

### PR DESCRIPTION
**Pull request by megbrown**
Related Issue:

Changes:
Fleshed out information

Readme Info:
 - [ ] required packages (new)
 - [ ] API instructions

Any other Testing Steps:
make sure accurate, look at formatting, make sure image works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,239 @@
 # Bangazon API
-Bangazon-corp's employees, products and users database
+This is Bangazon Co.'s employees, products and users API. Users of the API can GET/POST/PUT/DELETE information to and from the database for selected categories.
```
Comment (Arwask) [inline README.md]: Bangazon Corp sounds better than Co to me.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,239 @@
...
-## How to get started?
-- make your own .env file at the root of the project using .env.example file as a guide.
+## Installation
+- to clone the project down, run  ```git clone [repo link]```
+- run ```npm install``` from the root of the directory to install all of the dependencies
+
+
+## Get Started
+- set up the database using ```npm run db:reset``` command in terminal
+- create your own .env file at the root of the project using the .env.example file as a guide
```
Comment (Arwask) [inline README.md]: Do we want to explain what env is a little bit?

File: README.md [hunk 3]
```diff
@@ -1,20 +1,239 @@
...
+- run ```nodemon app.js``` from the terminal
+
+
+## Helper Applications
+- [DB Browser for SQLite](http://sqlitebrowser.org/) to work directly with database without command line
+- [Postman](https://www.getpostman.com/) to use PUT/POST/DELETE http http methods
```
Comment (Arwask) [inline README.md]: found the double http

File: README.md [hunk 4]
```diff
@@ -1,20 +1,239 @@
...
+1. [Get Started](#get-started)
+1. [Helper Applications](#helper-applications)
+1. [Usage Directions](#usage-directions)
+1. [Third Party Libraries](#third-party-libraries)
+1. [Credits](#credits)
+1. [Contribute to the API](#contribute-to-api)
+
+
+## Software Requirements
 - [NodeJs](https://nodejs.org/en/)
```
Comment (joshualloyd) [inline README.md]: Change spelling to "Node.js"

File: README.md [hunk 5]
```diff
@@ -1,20 +1,239 @@
...
+- run ```nodemon app.js``` from the terminal
+
+
+## Helper Applications
+- [DB Browser for SQLite](http://sqlitebrowser.org/) to work directly with database without command line
+- [Postman](https://www.getpostman.com/) to use PUT/POST/DELETE http http methods
+
+
+## Usage directions
+- go to ```api/v#/``` to view route and method layout, see additional details below
```
Comment (joshualloyd) [inline README.md]: path should be "/api/v1/"

**Review by joshualloyd — CHANGES_REQUESTED** [review]
found a couple things

File: README.md [hunk 6]
```diff
@@ -1,20 +1,239 @@
...
+  * you can use postman to do a PUT at given url
+  * you will need to use the keys from example body for all the fields you would like to edit
+  * route: ```api/v1/orders/1``` - edit order with order_id = 1
+
+* DELETE - will let you delete a specific chunk of data
+  * you can use postman to DELETE at given url
+  * route: ```api/v1/orders/1``` - deletes order with order_id = 1
+  > NOTE: Each Table's DELETE works differently depending on the entity relationships and dependencies. See specific table for detailed delete functionality.
+
+### **Employees**
```
Comment (Arwask) [inline README.md:67]: Remove bullets on all the table details

File: README.md [hunk 7]
```diff
@@ -1,20 +1,239 @@
...
 ## Third Party Libraries
-- Express
+- [express](https://www.npmjs.com/package/express)
+- [sqlite3](https://www.npmjs.com/package/sqlite3)
+- [nodemon](https://www.npmjs.com/package/nodemon)
+- [dotenv](https://www.npmjs.com/package/dotenv)
+- [body-parser-json](https://www.npmjs.com/package/body-parser-json)
+
+
+## Credits
```
Comment (Arwask) [inline README.md:210]: Seems sub headings are bolder than the main heading Credits. Does it look weird?

**Review by Arwask — CHANGES_REQUESTED** [review]
What do you think about the requested changes

File: README.md [hunk 8]
```diff
@@ -1,20 +1,239 @@
...
+	"seller_user_id": "INTEGER"
+}
+```
+
+### **Orders**
+- http methods supported: GET, POST, PUT, DELETE
+> NOTE: deleting an order will delete the associated entry/entries on the orderProduct join table
+- example body:
+```
+{
```
Comment (Arwask) [inline README.md]: need a product_id???

**Review by Arwask — APPROVED** [review]
Looks good to me

**Review by joshualloyd — APPROVED** [review]

**Pull request merged**